modules/statics/src/base.ts

```diff
@@ -649,6 +650,7 @@ export enum UnderlyingAsset {
   STBU = 'stbu',
   STC = 'stc',
   STCV2 = 'stcv2',
+  SETHH = 'seth-h',
```

Comment:
```suggestion
 'SETH-H' = 'seth-h',
```
Would that meet the constraints you are looking for in regards to the hyphen or are there too many down stream services that would need to change interpretations of the enum?

Reply:
Well, that's the main issue we dont know the full implications. But for XML is use your recomendations so I can change to use your suggestion.

Reply:
It's a new addition `statics` wise, and it's JavaScript so any service wanting to make use of it should be able to use it seamlessly.

Reply:
done